freqtrade/freqtradebot.py

```diff
@@ -269,10 +269,17 @@ def get_free_open_trades(self) -> int:
     def update_funding_fees(self):
         if self.trading_mode == TradingMode.FUTURES:
             for trade in Trade.get_open_trades():
-                funding_fees = self.exchange.get_funding_fees_from_exchange(
-                    trade.pair,
-                    trade.open_date
-                )
+                if self.config['dry_run']:
+                    funding_fees = self.exchange.calculate_funding_fees(
+                        trade.pair,
```

Comment:
@xmatthias I want to test that these are called, how can I add a few trades that will be returned when `Trade.get_open_trades()` is called?

Reply:
`create_mock_trades`?

Reply:
All right, well I think this is the last thing I have to do for this PR, and then maybe also `test_ccxt_compat`